tesseract_ros/tesseract_rosutils/src/plotting.cpp

```diff
@@ -50,7 +52,11 @@ ROSPlotting::ROSPlotting(std::string root_link, std::string topic_namespace)
   axes_pub_ = nh.advertise<visualization_msgs::MarkerArray>(topic_namespace + "/display_axes", 1, true);
 }
 
-bool ROSPlotting::init(tesseract::Tesseract::ConstPtr /*thor*/) { return true; }
+bool ROSPlotting::init(tesseract::Tesseract::ConstPtr thor)
```

Comment:
Curious what `thor` means. I noticed you started using it.

Reply:
I needed something instead of tesseract because it is also a namespace and I like Marvel [Thor](https://en.wikipedia.org/wiki/Thor). :)